tests/e2e/linalg_ext_ops/attention.mlir

```diff
@@ -12,3 +12,101 @@ func.func @attention() {
   ) : tensor<1x4x4xf32>
   return
 }
+
+func.func @attention1x3x4() {
+  %init = tensor.empty() : tensor<1x3x4xf32>
+  %query = util.unfoldable_constant dense<[[[0.1, 0.2, 0.3, 0.4],
+                                            [0.5, 0.6, 0.7, 0.8],
+                                            [0.9, 1.0, 1.1, 1.2]]]> : tensor<1x3x4xf32>
+
+  %key = util.unfoldable_constant dense<[[[0.1, 0.2, 0.3, 0.4],
+                                          [0.5, 0.6, 0.7, 0.8],
+                                          [0.9, 1.0, 1.1, 1.2]]]> : tensor<1x3x4xf32>
+  %value = util.unfoldable_constant dense<[[[0.1, 0.2, 0.3, 0.4],
+                                            [0.5, 0.6, 0.7, 0.8],
+                                            [0.9, 1.0, 1.1, 1.2]]]> : tensor<1x3x4xf32>
+  %scale = arith.constant 0.5 : f32
+  %1 = iree_linalg_ext.attention ins(%query, %key, %value, %scale : tensor<1x3x4xf32>,
+        tensor<1x3x4xf32>, tensor<1x3x4xf32>, f32) outs(%init : tensor<1x3x4xf32>) -> tensor<1x3x4xf32>
+  check.expect_almost_eq_const(
+      %1,
+      dense<[[[0.5530, 0.6530, 0.7530, 0.8530],
+              [0.6328, 0.7328, 0.8328, 0.9328],
+              [0.7011, 0.8011, 0.9011, 1.0011]]]> : tensor<1x3x4xf32>
+  ) : tensor<1x3x4xf32>
+  return
+}
+
+func.func @attention1x4x4() {
```

Comment:
Perhaps we can retire the first test case, since this covers the needs? What do you think?

Reply:
By the first test case, I meant `func.func @attention()`

Reply:
I agree with you. The names will make more sense, then.